docs/source/annotated_examples.rst

```diff
@@ -441,6 +386,34 @@ source: `scrollbar.py <https://github.com/enthought/chaco/tree/master/examples/d
 
 .. image:: example_images/scrollbar.png
 
+``simple_line.py``
+------------------
+Draws several overlapping line plots.
+
+Double-clicking on line or scatter plots opens a Traits editor for the plot.
+
+source: `simple_line.py <https://github.com/enthought/chaco/tree/master/examples/demo/simple_line.py>`_
+
+.. image:: ../images/simple_line.png
```

Comment:
this is broken at the moment, probably because the file got moved outside of the `user_manual`